docs/advanced-features/dynamic-import.md

```diff
@@ -156,3 +156,45 @@ function Home() {
 
 export default Home
 ```
+
+## With suspense
+
+In React 18, `<Suspense>` and `React.lazy` can work with SSR. So dynamic provide an option `suspense` to let you choose if you want to load a component under suspense.
+
+```jsx
+import { Suspense } from 'react'
+import dynamic from 'next/dynamic'
+
+const Profile = dynamic(() => import('./profile'), { suspense: true })
+
+function Home() {
+  return (
+    <div>
+      <Suspense fallback={'loading...'}>
+        <Profile />
+      </Suspense>
+    </div>
+  )
+}
+
+export default Home
+```
+
+If you set option `suspense` to true, other options will be omitted, delegating rendering control to React `<Suspense />`.
+It's similar to using `React.lazy` directly, but `next/dynamic` will support mode like CSR and SSG.
+
+To use `suspense`, you need to enable `reactRoot` and `concurrentFeatures` in `next.config.js`.
+
+Note: `reactRoot` will be enabled by default when using React 18.
+
+```jsx
+// next.config.js
+module.exports = {
+  experimental: {
+    reactRoot: true,
+    concurrentFeatures: true,
+  },
+}
+```
+
+If you only enable `reactRoot` but not `concurrentFeatures`, suspense will always render fallbacks on server side.
```

Comment:
I wish the docs had a way to mark something as experimental. I would kind of rather this not be documented for now...